NCLL/licence-area.html

```diff
@@ -184,9 +184,9 @@ <h1 property="name" id="wb-cont">Licence area</h1>
 						<li>Drawing the boundary on the interactive map</li>
 						<li>Entering a description of the boundary in <abbr>WKT</abbr> (Well Known Text) format into the input field below the map</li>
 					</ol>
-					<p>This application form is not designed to assist with network planning. If you would like to identify the boundary of an area that will not cause interference with other operators, please consider using our <a href="SLAB-en-leaflet.html" target="_blank">Spectrum licence area browser</a>.</p>
 					<p>Remember that you will need to be able to cover the full licence area. And because low power licences have a better chance of being approved, you may want to limit the size of your licence area to 15 km<sup>2</sup> or less.</p>
```

Comment:
We should clean up this sentence too, maybe make it into bullet points. The tone is a touch too casual.